.github/workflows/samples_connections.yml

```diff
@@ -78,7 +78,6 @@ jobs:
           subscription-id: ${{secrets.AZURE_SUBSCRIPTION_ID}}
           tenant-id: ${{secrets.AZURE_TENANT_ID}}
           client-id: ${{secrets.AZURE_CLIENT_ID}}
-          enable-AzPSSession: true
```

Comment:
add PR description on why this is removed.

Reply:
Samples failed when azure login: https://github.com/microsoft/promptflow/actions/runs/9406919010/job/25911489619

Offline discuss with @crazygao, maybe it related with powershell issues and remove the "enable-AzPSSession: true"